src/Option_Command.php

```diff
@@ -569,7 +569,13 @@ function( $key ) {
 				: null;
 			$patch_value = ! empty( $stdin_value )
 				? WP_CLI::read_value( $stdin_value, $assoc_args )
-				: WP_CLI::read_value( array_pop( $key_path ), $assoc_args );
+				: ( count( $key_path ) > 1
+					? WP_CLI::read_value( array_pop( $key_path ), $assoc_args )
+					: null );
```

Comment:
This nested ternary is a bit complex to read 😕

Can we expand it to some if/else statements?